### ISSUE TITLE
Need more wallpapers, more customizable wallpapers

### ISSUE THREAD

**Issue by Ncupst** [kiwibrowser/src.next#666]
Need more wallpapers, more customizable wallpapers

**Comment by Ultimatinium** [comment]
For newtab page..?

**Comment by BEEDELLROKEJULIANLOCKHART** [comment]
I disagree. Ideally, this software should not include any photography unless necessary because it increases the size of it as much as it does. For what you desire, @Ncupst, does a file-picker exist?
